airbyte-integrations/connectors/source-github/source_github/schemas/workflow_runs.json

```diff
@@ -526,6 +423,12 @@
           "type": ["null", "string"]
         }
       }
+    },
+    "actor": {
+      "$ref": "user.json"
+    },
+    "triggering_actor": {
+      "$ref": "user.json"
```

Comment:
I see that in the repo, there is a `users.json` schema for the users stream. I don't see a `user.json` anywhere - is this ref correct? (Used many other places too)

Reply:
all referenced sub-schemas located here `schemas/shared`

Reply:
Aha, that's what I was missing! Thanks